Functions-DOM/js-functions_dom-manipulation/index.html

```diff
@@ -15,7 +15,7 @@
       <h1>The magic colorful box</h1>
     </header>
     <main>
-      <div data-js="box" class="box gray"></div>
+      <div data-js="box" class="box"></div>
```

Comment:
Why did you manually remove the gray from the index.html? Ws supposed to be not touched.